frontend/src/components/OptionSelector.tsx

```
@@ -0,0 +1,28 @@
+}: OptionSelectorProps<T>) => {
+  return (
+    <div className="flex flex-wrap justify-center gap-4 max-w-lg">
+      {options.map((option) => (
+        <button
+          key={option}
+          onClick={() => onSelect(option)}
+          className={`px-4 py-2.5 rounded-full border text-sm font-medium transition-colors
+            ${selectedOption === option
+              ? 'bg-[#215E6B] text-white border-[#215E6B]'
+              : 'bg-white text-[#49454F] border-[#CAC4D0] hover:bg-gray-50'}`}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)